src/client/pages/VergunningDetail/VergunningDetail.tsx

```diff
@@ -64,6 +66,14 @@ export default function VergunningDetail() {
             {Vergunning.caseType === 'Evenement melding' && (
               <EvenementMelding vergunning={Vergunning} />
             )}
+            {Vergunning.caseType ===
```

Comment:
Zijn de caseTypes niet BZP en BZB ?

Reply:
BZP en BZB zijn de afkortingen voor Parkeerontheffingen Blauwe zone particulieren en Parkeerontheffingen Blauwe zone bedrijven. Het leek me handig deze als variabelnaam voor de types te gebruiken. Eens? In dat geval zal ik de file names ook zo noemen.

Reply:
Nee, alleen als het ook de `zaaktypes` zijnm dat is een beetje de conventie.

Reply:
In dat geval: ParkeerontheffingenBlauweZoneBedrijven ?

Reply:
Mja, dat zou dan kunnen.. maar ik zie waarom je die keuze niet hebt gemaakt. In een ander document is er wel sprake van BZB en BZP. We kunnen er voor kiezen om af te wijken van dat zaaktype maar ik wil niet een situatie waarbij we voor alles afkortingen gaan verzinnen. De interfaces worden nu ook BZB / BZP genoemd natuurlijk. 

Mja, ik zou zeggen. Neem BZP en BZB als uitgangspunt en zet een comment bij de interfaces mbt de keuze voor een afkorting ipv het volledig uitschrijven van de zaaktypes. 

Evt kunnen we ook eea refactoren en de zaaktypes mbh constants of enums gaan matchen. Bv

```
enum CaseType {
  BZB: 'Parkeerontheffingen blauwe zone bedrijven',
  .. etc,
  EvenementMelding: 'Evenement melding',
  .. etc
}

interface Vergunning {
  caseType: CaseType;
}
```

óf

```
const caseTypes = {
   BZB: 'Parkeerontheffingen blauwe zone bedrijven',
  .. etc,
  EvenementMelding: 'Evenement melding',
  .. etc
}
```

en

```
if(vergunning.caseType === CaseType.BZB) { ... } 
```

Reply:
Wat denk jij?

Reply:
Ik ben wel fan van de oplossing met de casetypes refactoren naar enums. Ik denk dat dit sowieso een goede toevoeging is. Alleen het is nog niet helemaal een antwoord op de vraag: wel of geen afkorting gebruiken? In mijn optiek is het prima om een afkorting te gebruiken. Al ben ik het met je eens om er wel voorzichtig mee te zijn en het niet voor elke habbekrats te doen.

Reply:
Ja precies, daarom die comment bij de interfaces.